speedy-recovery-client/src/service/FhirMappingAdapterSTU3.js

```diff
@@ -14,6 +14,7 @@ import {
 const missingField = "Unknown";
 const mapPatientToUserSTU3 = fhirPatientResource => ({
   //This is a temporary hard-code fix as we have not implemented the searching for a patients' parent
+  id: fhirPatientResource.id ? fhirPatientResource.id : "Unknown",
```

Comment:
To be consistent, can we use the missingField variable instead of explicitly outputting "Unkown"